Jellyfin.Server/StartupOptions.cs

```diff
@@ -73,6 +73,12 @@ public class StartupOptions : IStartupOptions
         [Option("nonetchange", Required = false, HelpText = "Indicates that the server should not detect network status change.")]
         public bool NoDetectNetworkChange { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the server should not detect folders that are merged by configuration.
+        /// </summary>
+        [Option("nocrossaccesscheck", Required = false, HelpText = "Indicates that the server should not detect overlapping folder.")]
+        public bool NoFolderCrossCheck { get; set; }
```

Comment:
I don't really see a good use for this option. We know that the server will (horribly) break with the overlapping paths so why even allow a user to ignore that?

Reply:
i _can_ see usecases where though some docker magic you _could_ non-destructivly put for example your config and metadata path under the same root. Its just a fallback for people that do crazy shit (or also LSIO containers i have not tested).

Reply:
But do we really want to support that niche?

Reply:
well i see no harm in offering a simple solution if we break something with that. Its not complicated nor hard to maintain. But if the concenses is against that, i have no issue with removing the toggle.

Reply:
IMO we should break/discourage setups that are asking for trouble, but maybe it's not super user friendly now but better in the long run. And in this case I don't think we can prevent some of the scheduled tasks for temp files clobbering say ones config, right? Better to just have people not be stupid.

Reply:
I would still argue to leave this in __for now__. Its fine if we break those users installs (if they are working) but not without warning and time to do it right for as long as we can.